features/submitHealthEvent.feature

```diff
@@ -18,3 +18,50 @@ Feature: Submit Health Event
      When I receive a response
      Then I expect response should have a status 422
       And I expect response should validate against the profile http://hl7.org/fhir/StructureDefinition/OperationOutcome
+
+  # Language-specific Scenarios
+
```

Comment:
Right now, we don't really use the tags for anything, so I don't know if this necessarily provides any benefit. However I think there could be benefit to each test having it's own unique tag, so adding the @SubmitValidHealthEvent to the other language scenarios might convolute that tag a bit. 

It also could be a bit of a slippery slope. If we group all of the Valid SubmitHealthEvent transactions together, then we would also probably want to have an overarching @StoreInvalidHealthCertificate tag that encompasses all of the different Error scenarios there, etc. 

I personally think this might add complexity for no real benefit, but I'm open to other opinions.

Reply:
I did just push a commit that changes the names of the language-specific SubmitHealthEvent transactions to clarify that they are also valid health events.